src/Stripe.Tests.XUnit/Stripe.Tests.XUnit.csproj

```diff
@@ -27,4 +27,8 @@
     <ProjectReference Include="..\Stripe.net\Stripe.net.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
Do you know why these ended up in here? Can we pull them out of the branch for hygiene?